src/rebalancers/Rebalancer.sol

```diff
@@ -0,0 +1,484 @@
+            // We do not handle the edge cases where the new ticks might exceed MIN_TICK or MAX_TICK.
+            // This will result in a revert during the mint, if ever needed a different rebalancer has to be deployed
+            tickCurrent = tickCurrent / position.tickSpacing * position.tickSpacing;
+            // For tick ranges that are an even multiple of the tick spacing, we use a symmetric spacing around the current tick.
+            // For uneven multiples, the smaller part is below the current tick.
+            position.tickLower = tickCurrent - tickRange / (2 * position.tickSpacing) * position.tickSpacing;
+            position.tickUpper = position.tickLower + tickRange;
+        } else {
+            (position.tickLower, position.tickUpper) = (tickLower, tickUpper);
+        }
+        position.sqrtRatioLower = TickMath.getSqrtRatioAtTick(position.tickLower);
+        position.sqrtRatioUpper = TickMath.getSqrtRatioAtTick(position.tickUpper);
+
+        // Get trusted USD prices for 1e18 gwei of token0 and token1.
+        (uint256 usdPriceToken0, uint256 usdPriceToken1) =
+            ArcadiaLogic._getValuesInUsd(position.token0, position.token1);
+
+        // Calculate the square root of the relative rate sqrt(token1/token0) from the trusted USD price of both tokens.
+        uint256 trustedSqrtPriceX96 = PricingLogic._getSqrtPriceX96(usdPriceToken0, usdPriceToken1);
+
+        // Calculate the upper and lower bounds of sqrtPriceX96 for the Pool to be balanced.
+        // We do not handle the edge cases where exceed MIN_SQRT_RATIO or MAX_SQRT_RATIO.
+        // This will result in a revert during swapViaPool, if ever needed a different rebalancer has to be deployed.
+        position.lowerBoundSqrtPriceX96 =
+            trustedSqrtPriceX96.mulDivDown(initiatorInfo[initiator].lowerSqrtPriceDeviation, 1e18);
+        position.upperBoundSqrtPriceX96 =
+            trustedSqrtPriceX96.mulDivDown(initiatorInfo[initiator].upperSqrtPriceDeviation, 1e18);
+    }
+
+    /* ///////////////////////////////////////////////////////////////
+                            ACCOUNT LOGIC
+    /////////////////////////////////////////////////////////////// */
+    /**
+     * @notice Sets the required information for an Account.
+     * @param account_ The contract address of the Arcadia Account to set the information for.
+     * @param initiator The address of the initiator.
+     * @param hook The contract address of the hook.
+     * @dev An initiator will be permissioned to rebalance any
+     * Liquidity Position held in the specified Arcadia Account.
+     * @dev If the hook is set to address(0), the hook will be disabled.
+     * @dev When an account is transferred to a new owner,
+     * the asset manager itself (this contract) and hence its initiator and hook will no longer be allowed by the Account.
+     */
+    function setAccountInfo(address account_, address initiator, address hook) external {
```

Comment:
Just thinking if would make sense here to allow an Asset Manager to call setAccountInfo() on behalf of the Account.
As I can imagine an Asset Manager at higher level that part of it's strategy would use combinations of rebalancers/strategyHooks (but maybe a bit early for this).